bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_utils.py

```diff
@@ -74,6 +81,23 @@ def _string_to_root_attribute(value, lookup_map):
         return value
 
 
+def _dict_to_std_map(arg_dict, template_arg):
+    """Helper function to convert python dict to std::map."""
+    import ROOT
+
+    # The std::map created by this function usually contains non-owning pointers as values.
+    # This is not considered by Pythons reference counter. To ensure that the pointed-to objects
+    # live at least as long as the std::map, a python list containing references to these objects
+    # is added as an attribute to the std::map.
+    arg_map = ROOT.std.map[template_arg]()
+    arg_map.keepalive = list()
+    for key, value in arg_dict.items():
+        arg_map.keepalive.append(value)
+        arg_map.emplace(key, value)
```

Comment:
I believe this function should do a check of the types of the keys and values of the dictionary you are about to convert. If they don't all match the types of the template argument, we should error out in a meaningful way to the user, saying that they didn't provide the right types in the dictionary (here we would just crash because the `emplace` operation would fail due to the types of its arguments).

Reply:
I think it's okay as it is, because we check that everything is the correct type before in the function.

However, I still get linker errors for some reason with this emplace :) Can you use the bracket operator instead, please? This works for me:
```Python
arg_map[key] = value
```